app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(team: list[Player]) -> int:
+    result = 0
+    for member in team:
+        result += member.get_rating()
+
+    return result
```

Comment:
You could write this part in one line using an example [from the checklist](https://github.com/IlyaGrynyshyn/py-elves-and-dwarves/blob/develop/checklist.md#code-efficiency).